QRealTime_dialog.py

```diff
@@ -624,6 +620,82 @@ def getTable(self,XFormKey,importData,topElement,version= 'null'):
         except Exception as e:
             print ('not able to fetch',e)
             return response,table
+
+class Central (Aggregate):

```

Comment:
I believe you should inherit from kobo CLASS as it has more similarity to ODK Central